worldedit-core/src/main/java/com/sk89q/worldedit/internal/expression/invoke/ExpressionCompiler.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * WorldEdit, a Minecraft world manipulation toolkit
+ * Copyright (C) sk89q <http://www.sk89q.com>
+ * Copyright (C) WorldEdit team and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.sk89q.worldedit.internal.expression.invoke;
+
+import com.google.common.collect.SetMultimap;
+import com.sk89q.worldedit.antlr.ExpressionParser;
+import com.sk89q.worldedit.internal.expression.CompiledExpression;
+
+import java.lang.invoke.LambdaConversionException;
+import java.lang.invoke.LambdaMetafactory;
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodType;
+
+import static com.sk89q.worldedit.internal.expression.invoke.ExpressionHandles.COMPILED_EXPRESSION_SIG;
+import static com.sk89q.worldedit.internal.expression.invoke.ExpressionHandles.safeInvoke;
+import static java.lang.invoke.MethodType.methodType;
```

Comment:
These static imports don't help readability.
I think resolving those would help one to visually spot that, for example, COMPILED_EXPRESSION_SIG is from another class.

Reply:
I guess the question is how many of them to remove? I can understand removing `COMPILED_EXPRESSION_SIG`, but `safeInvoke` gets a little verbose without the import.

`MethodType.methodType` is just plain redundant, and I don't agree with removing that import.

Reply:
safeInvoke seems to be only used once in this class

Reply:
Yea, I inlined it for this class.